crawl-ref/source/dat/des/serial/corridors.des

```diff
@@ -159,3 +159,59 @@ xxxxxxxx...xxxxxxxx
       xG...Gx
       xx@@@xx
 ENDMAP
+
+NAME:   chequers_hall_statues_y
+DEPTH:  D, Depths, Elf, Crypt, Snake
+WEIGHT: 2
+TAGS:   transparent extra allow_dup serial_corexii_hall decor
+SUBST:  T = T:3 V:3 G:1 .:3
+MAP
+         x
+         x
+xx@@@xxxxxxxxx@@@xx
+xG....GxxxxxG....Gx
+xx.....xxxxx.....xx
+ xx.....xxx.....xx
+  xx.....x.....xx
+   xG.........Gx
+    xx.......xx
+     xx.....xx
+      xx...xx
+      xG...Gx
+      xx...xx
+      xx...xx
+      xx...xx
+      xG...Gx
+      xx@@@xx
+ENDMAP
+
+NAME:   chequers_hall_statues_wall_a
+DEPTH:  D, Depths, Elf, Crypt, Snake
+WEIGHT: 2
+TAGS:   transparent extra allow_dup serial_corexii_hall decor
+SUBST:  T = T:3 V:3 G:1 .:3
+MAP
+xxxxxxxxxxxxxxxxxxx
+xGxxxGxxxGxxxGxxxGx
+@.................@
+@.................@
+ENDMAP
+
+NAME:   chequers_hall_statues_corner_a
+DEPTH:  D, Depths, Elf, Crypt, Snake
+WEIGHT: 2
+TAGS:   transparent extra allow_dup serial_corexii_hall decor
+SUBST:  T = T:3 V:3 G:1 .:3
+MAP
+xxxxxxxxxxx
+xGxxxGxxxGx
+xx........@
+xx........@
+xx..
```

Comment:
Again, the other serial_corexii_hall vaults have enclosed corridors, so having this one open seems weird. If you're doing this on purpose, why?

Reply:
It's a decor vault that just adds an open L-shaped wall. The corridor notion is about connecting two sides of the vaults to the rest of the level. The decor vaults in this serial don't have to be fully enclosed corridors; they're trying to add some additional embellishment to structure normally generated by the layout (which is a pretty open layout when you look at levels it generates).